apps/software/lib/software/model/storage_drive.ex

```diff
@@ -16,6 +16,7 @@ defmodule Helix.Software.Model.StorageDrive do
   @creation_fields ~w/storage_id/a
 
   @primary_key false
+  @ecto_autogenerate {:drive_id, {PK, :pk_for, [__MODULE__]}}
```

Comment:
FK to component_id (well, not exactly because it is a different database but you got the idea)